File: go/wam/2746.go
const WamMdCriticalEvent = 2746 // channel: regular

type MdCriticalEventEvent struct {
	MdCriticalEventCode MDSYNCDCRITICALEVENTCODE `wam:"md_critical_event_code"` // field 1
	Collection COLLECTION `wam:"collection"` // field 2
	MutationActionName string `wam:"mutation_action_name"` // field 3
	MdCriticalEventErrorMessage string `wam:"md_critical_event_error_message"` // field 4
	MdCriticalEventStage string `wam:"md_critical_event_stage"` // field 5
}

File: ts/wam/2746.ts
export const WamMdCriticalEvent = 2746 as const

export interface MdCriticalEventEvent {
  /** field 1, wire `md_critical_event_code` */
  mdCriticalEventCode?: MDSYNCDCRITICALEVENTCODE
  /** field 2, wire `collection` */
  collection?: COLLECTION
  /** field 3, wire `mutation_action_name` */
  mutationActionName?: string
  /** field 4, wire `md_critical_event_error_message` */
  mdCriticalEventErrorMessage?: string
  /** field 5, wire `md_critical_event_stage` */
  mdCriticalEventStage?: string
}